message/message_test.go

```diff
@@ -81,3 +82,41 @@ func TestOpenClose(t *testing.T) {
 		return
 	}
 }
+
+// Test we can reply to the rpcin
+func TestReset(t *testing.T) {
```

Comment:
Does this test pass if vmtoolsd is running?  May want to skip this one in any case, unless an env var or flag is set.